tests/virt_control_plane_test.go

```diff
@@ -233,48 +231,31 @@ var _ = Describe("[Serial][ref_id:2717][sig-compute]KubeVirt control plane resil
 
 		When("Control plane pods temporarily lose connection to Kubernetes API", func() {
 			// virt-handler is the only component that has the tools to add blackhole routes for testing healthz. Ideally we would test all component healthz endpoints.
-			componentName := "virt-handler"
+			const componentName = "virt-handler"
 
-			getVirtHandler := func() *v1.DaemonSet {
-				daemonSet, err := virtCli.AppsV1().DaemonSets(flags.KubeVirtInstallNamespace).Get(context.Background(), componentName, metav1.GetOptions{})
-				ExpectWithOffset(1, err).NotTo(HaveOccurred())
-				return daemonSet
-			}
-
-			readyFunc := func() int32 {
-				return getVirtHandler().Status.NumberReady
-			}
-
-			blackHolePodFunc := func(addOrDel string) {
+			handlerPodGetter := func(virtCli kubecli.KubevirtClient, componentName, namespace, nodeName string) *k8sv1.PodList {
```

Comment:
This adds an unnecessary closure. If we really need it, please add it as separate func to this file.

Reply:
I've better used the closures as we discussed.